### ISSUE TITLE
PyPI sdist is missing `requirements.txt`

### ISSUE THREAD

**Issue by p-linnane** [gnachman/gptline#2]
Hello. I'm packaging gptline for distribution via Homebrew. Attempting to build from the sdist on PyPI fails due to a missing `requirements.txt` file referenced in `setup.py`.

```shell
==> python3.11 -m pip --python=/opt/homebrew/Cellar/gptline/1.0.7/libexec/bin/python install --verbose --no-deps --no-binary=:all: --ignore-installed --no-compile /private/tmp/gptline-20231005-61473-ogkkjf/gptline-1.0.7
Using pip 23.2.1 from /opt/homebrew/lib/python3.11/site-packages/pip (python 3.11)
Processing /private/tmp/gptline-20231005-61473-ogkkjf/gptline-1.0.7
  Preparing metadata (setup.py): started
  Running command python setup.py egg_info
  Traceback (most recent call last):
    File "<string>", line 2, in <module>
    File "<pip-setuptools-caller>", line 34, in <module>
    File "/private/tmp/gptline-20231005-61473-ogkkjf/gptline-1.0.7/setup.py", line 3, in <module>
      with open("requirements.txt") as f:
           ^^^^^^^^^^^^^^^^^^^^^^^^
  FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
```

**Comment by gnachman** [comment]
That's odd, it's right here:

https://github.com/gnachman/gptline/blob/main/requirements.txt

**Comment by p-linnane** [comment]
Yeah it somehow isn't being included when publishing to PyPI.

```
╰─ tar -vxzf gptline-1.0.7.tar.gz
x gptline-1.0.7/
x gptline-1.0.7/LICENSE
x gptline-1.0.7/PKG-INFO
x gptline-1.0.7/README.md
x gptline-1.0.7/gptline.egg-info/
x gptline-1.0.7/gptline.egg-info/PKG-INFO
x gptline-1.0.7/gptline.egg-info/SOURCES.txt
x gptline-1.0.7/gptline.egg-info/dependency_links.txt
x gptline-1.0.7/gptline.egg-info/entry_points.txt
x gptline-1.0.7/gptline.egg-info/requires.txt
x gptline-1.0.7/gptline.egg-info/top_level.txt
x gptline-1.0.7/setup.cfg
x gptline-1.0.7/setup.py
x gptline-1.0.7/src/
x gptline-1.0.7/src/__init__.py
x gptline-1.0.7/src/background_task.py
x gptline-1.0.7/src/chat.py
x gptline-1.0.7/src/db.py
x gptline-1.0.7/src/formatting.py
x gptline-1.0.7/src/highlight.py
x gptline-1.0.7/src/input_reader.py
x gptline-1.0.7/src/main.py
x gptline-1.0.7/src/search.py
x gptline-1.0.7/src/spin.py
x gptline-1.0.7/src/ui_utils.py
```

**Comment by gnachman** [comment]
Fixed in version 1.0.8.
